adapter/noop2/noop2.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 Istio Authors.
```

Comment:
Okie, assuming the ConfigureMetric, ConfigureQuota etc. methods can be called after the adapter is atomically created, I have added a builder that will get registerted, which we will call `Create` on, passing the proto.Message adapter config, to instantiate an Adapter instance. PTAL at the change.

The name Handler and Adapter are confusing me. From the Adapter implementation author's point of view, what is a Handler ?